### PR TITLE
version bump and Changes update

### LINKED ISSUE #1653: Haraka release 2.8.10
- [ ] Start a N-week quiet period
- [x] Update [haraka-tld](https://github.com/haraka/haraka-tld)
- [x] Draft a [release](https://github.com/haraka/Haraka/releases/tag/untagged-46246600315d079ae0b3)
- Create a PR with:
    - [x] Updated [Changes](https://github.com/haraka/Haraka/blob/master/Changes)
    - [x] Bumped version in [package.json](https://github.com/haraka/Haraka/blob/master/package.json)
    - [x] Updated dependencies identified with `grunt versioncheck`
- [ ] Update release date in [Changes](https://github.com/haraka/Haraka/blob/master/Changes)
- [ ] `npm publish`
- [ ] Post `Changes` file on the draft release and publish the release 
- [ ] update [haraka's wikipedia page](https://en.wikipedia.org/wiki/Haraka_(software))
- [ ] Email the mailing list
- [ ] Publish a blog post
- [ ] Edit the /topic on #haraka IRC channel

### PR DESCRIPTION
**Pull request by msimerson**
Fixes #1653 

Changes proposed in this pull request:
- update package.json
- update Changes

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,38 +1,67 @@
-2.8.9 - Oct 02, 2016
+2.8.10 - Oct 1?, 2016
```
Comment (godsflaw) [inline Changes]: did you mean to put `?` for the day here?

Comment (msimerson) [inline Changes]: Yeah, it will get updated when this version gets released.

**Comment by godsflaw** [comment]
+1

**Comment by msimerson** [comment]
I'm deploying haraka/Haraka#master to my server now. My intent is to get 2.8.10 published today, barring any unforeseen issues or objections.

**Comment by msimerson** [comment]
````
$ npm publish
+ Haraka@2.8.10
````

**Pull request merged**